### ISSUE TITLE
BackinStore object should be deleted if it is deleted from noobaa

### ISSUE THREAD

**Issue by rgordill** [noobaa/noobaa-operator#249]
Hi.

When deleting a backinstore from Noobaa UI, it is not deleted from kubernetes, and it shows ready status on the summary.

Steps to reproduce:

- Create a BackinStore object through k8s (kubectl create -f backing-store.yaml)
- Wait until it is created on Noobaa
- Open Noobaa console, go to resources and delete it

BackingStore object is still present:

```
> kubectl get backingstore.noobaa.io/quay
NAME   TYPE      PHASE   AGE
quay   pv-pool   Ready   12m
```

**Comment by guymguym** [comment]
Hi @rgordill Can you provide the operator version you are using?

**Comment by rgordill** [comment]
Latest one: noobaa-linux-v2.0.10

**Comment by guymguym** [comment]
Thanks @rgordill! 

Issue #164 was about propagating the status back to the backing store status.
It was fixed for the 2.1 branch, but not going to be backported to the 2.0 branch as the fix is a bit too involved. So at least the status propagation will work once we make the first 2.1 release...

Now regarding deleting the backing-store itself when deleted from the UI - this is the same as issue #137 and is still open.

**Comment by rgordill** [comment]
Ok, closing this issue.

Duplicate of #137 and #164
